src/tools/mds_client.cpp

```diff
@@ -57,17 +64,6 @@ int MDSClient::Init(const std::string& mdsAddr,
             std::cout << "Init channel to " << mdsAddr << "fail!" << std::endl;
             continue;
         }
-        // 寻找哪个mds存活
-        curve::mds::topology::ListPhysicalPoolRequest request;
-        curve::mds::topology::ListPhysicalPoolResponse response;
-        curve::mds::topology::TopologyService_Stub stub(&channel_);
-        brpc::Controller cntl;
-        cntl.set_timeout_ms(FLAGS_rpcTimeout);
-        stub.ListPhysicalPool(&cntl, &request, &response, nullptr);
-
-        if (cntl.Failed()) {
-            continue;
-        }
         currentMdsIndex_ = i;
         isInited_ = true;
```

Comment:
The logic here should be to select the leader of mds, and currentMdsIndex_ should be the index of the leader mds

Reply:
It will try next if the first mds is not leader and previous logic is based send ListPhysicalPoolRequest, but I need add auth key with tools to mds before everything created.